app/src/main/java/de/danoeh/antennapod/fragment/ItemlistFragment.java

```diff
@@ -618,26 +618,22 @@ private void loadItems() {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe(result -> {
-                    feed = result;
+                    feed = result.orElse(null);
                     itemsLoaded = true;
                     if (viewsCreated) {
                         onFragmentLoaded();
                     }
                 }, error -> Log.e(TAG, Log.getStackTraceString(error)));
     }
 
-    @Nullable
-    private Feed loadData() {
+    private Optional<Feed> loadData() {
         Feed feed = DBReader.getFeed(feedID);
-        if (feed == null) {
-            return null;
-        }
-        DBReader.loadAdditionalFeedItemListData(feed.getItems());
-        if (feed.getItemFilter() != null) {
+        if(feed != null && feed.getItemFilter() != null) {
```

Comment:
Please add a space between `if` and `(`

Reply:
Doh, this was probably due to a merge conflict I had. Good catch!